api/Dockerfile

```
@@ -102,6 +102,9 @@ COPY composer.json composer.lock symfony.lock ./
 # (for docker-compose version, we need a least on file to exist, so we add the composer.json to prevent failure if packages is missing)
 COPY composer.json package* packages/
 
+ARG COMPOSER_AUTH_ARG
```

Histoire d'aligner la commande que tu lances:

```suggestion
ARG COMPOSER_AUTH
ENV COMPOSER_AUTH="${COMPOSER_AUTH}"
```